lib/v2/591/list.js

```diff
@@ -0,0 +1,211 @@
+const got = require('@/utils/got');
+const { CookieJar } = require('tough-cookie');
+const cheerio = require('cheerio');
+
+const cookieJar = new CookieJar();
+
+const client = got.extend({
+    cookieJar,
+    headers: {
+        'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.114 Safari/537.36',
+    },
```

Comment:
RSSHub will use Chrome on Mac UA by default.
https://github.com/DIYgod/RSSHub/blob/3195eb86f342aaa5ca978092633d1dea8ded465b/lib/config.js#L45

Is there any specific to use this UA over the default one?

Reply:
Thanks! I didn't know that, I just move my code from other places here. I'll test it to see if it works. 👍